test/xpu/xpu_test_utils.py

```diff
@@ -143,10 +151,6 @@
     "addr",
     "cdist",
     "nn.functional.group_norm",
-    "nn.functional.batch_norm",
-    "native_batch_norm",
-    "_native_batch_norm_legit",
-    "_batch_norm_with_update",
```

Comment:
Why skip them? They are supported operators listed in op_db?
<img width="679" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/94b24cf8-a0a7-46d0-afa4-6e70ff0f1406">

Reply:
revert it